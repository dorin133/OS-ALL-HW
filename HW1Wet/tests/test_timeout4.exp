smash> smash> smash> [1] timeout 3 sleep 2& : 111002 0 secs
smash> smash: got an alarm
smash> smash: sending SIGKILL signal to 0 jobs:
