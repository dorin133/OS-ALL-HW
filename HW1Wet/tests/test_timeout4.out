smash> smash> smash> [1] timeout 3 sleep 2& : 111003 0 secs
smash> smash: got an alarm
smash> smash: sending SIGKILL signal to 0 jobs:
